### DISCUSSION TITLE
2.28.0

### DISCUSSION THREAD

**Discussion by Jolg42** [prisma/prisma#8459] [Q&A]
Today, we are excited to share the `2.28.0` stable release 🎉

🌟 **Help us spread the word about Prisma by starring the repo ☝️ or [tweeting](https://twitter.com/intent/tweet?text=Check%20out%20the%20latest%20@prisma%20release%20v2.28.0%20%F0%9F%9A%80%0D%0A%0D%0Ahttps://github.com/prisma/prisma/releases/tag/2.28.0) about the release.** 🌟 

### MongoDB improvements 🚀

Thanks to your feedback, we fixed a handful of bugs reported on **the MongoDB connector (Preview)**:

- Concurrent `findUnique` queries leading to an error [#8276](https://github.com/prisma/prisma/issues/8276)
- Filtering by relations wasn't working properly [#7057](https://github.com/prisma/prisma/issues/7057)
- Filtering on array of IDs [#6998](https://github.com/prisma/prisma/issues/6998)

Please keep reporting issues to our team and help to bring MongoDB support closer to GA!

### Create new Prisma projects in under 3 minutes ⏳

With our latest releases in the [Application Data Platform](https://cloud.prisma.io/) you can now create new Prisma projects which contain the following:

- A GitHub repository in your connected GitHub account which has a Prisma schema in it.
- You can choose that schema from a selection of our templates: a URL shortener app, a Music streaming service, a Rentals platform, a SaaS app.
- A free database from Heroku, if you have an account there.
- As with every Prisma project, you get an online Databrowser which comes with authentication and User roles for different permission setups.

This whole process now takes less than 3 minutes to set up, so we're looking forward to seeing how you will use that feature, for your prototyping or production needs.

If you have any issues or questions, let us know by [opening a GitHub issue](https://github.com/prisma/studio/issues?q=is%3Aopen+is%3Aissue+label%3A%22topic%3A+hosted+data+browser%22).

![Prisma cloud onboarding](https://user-images.githubusercontent.com/27310414/127045405-b2d6f4c3-5970-439f-9c8f-4f6d534b10c4.png)

If you have a Heroku account, we can create a free Postgres database for you

![schema_templates](https://user-images.githubusercontent.com/27310414/127045541-ba5df553-bfc5-4d6f-be5f-f2f18abe626f.png)

Start your project with a schema from our templates.

### Interested in Prisma’s upcoming Data Proxy for serverless backends? Get notified! 👀

Database connection management in serverless backends is challenging: taming the number of database connections, additional query latencies for setting up connections, etc.

At Prisma, we are working on a Prisma Data Proxy that makes integrating traditional relational and NoSQL databases in serverless Prisma-backed applications a breeze. If you are interested, you can sign up to get notified of our upcoming Early Access Program here:

[https://pris.ly/prisma-data-proxy](https://pris.ly/prisma-data-proxy)

### Prisma Client

- [PANIC in query-engine/connectors/mongodb-query-connector/src/value.rs:100:24 not yet implemented: (ObjectId, List([String("609675d400e7693e0090e48c")]))](https://github.com/prisma/prisma/issues/6998)
- [MongoDB: Filtering by relation returns no results](https://github.com/prisma/prisma/issues/7057)
- [MongoDB: 2 queries for findUnique lead to error](https://github.com/prisma/prisma/issues/8276)

### Prisma Migrate

- [Using render.com database for `migrate dev` leads to error message `ERROR: cannot drop view pg_buffercache because extension pg_buffercache requires it HINT: You can drop extension pg_buffercache instead.`](https://github.com/prisma/prisma/issues/8217)

### Prisma Studio

- [Can't open Prisma Studio from WSL 2](https://github.com/prisma/studio/issues/719)
- [Unable to get DMMF from Prisma Client](https://github.com/prisma/studio/issues/720)
- [Prisma Studio fails to add or update record in database when it contains `"`.](https://github.com/prisma/studio/issues/722)
- [Scrolling & searching in initial schema list not working](https://github.com/prisma/studio/issues/723)
- [Fatal error on latest version (0.410.0) on Windows](https://github.com/prisma/studio/issues/725)
- [Ability to remove invalid/unused schemes](https://github.com/prisma/studio/issues/726)
- [fields with @updatedAt are not behaving as expected](https://github.com/prisma/studio/issues/728)
- [Error starting Prisma Client: {   "message": "\n    Error: spawn typegraphql-prisma ENOENT\n        at notFoundError ](https://github.com/prisma/studio/issues/730)
- [prisma studio error after installing latest version](https://github.com/prisma/studio/issues/732)
- [Sort models alphabetically (or allow multiple sorting options)](https://github.com/prisma/studio/issues/748)

## Credits

Huge thanks to @ShubhankarKG, @hehex9 for helping!

## 📺 Join us for another "What's new in Prisma" livestream

Learn about the latest release and other news from the Prisma community by joining us for another ["What's new in Prisma"](https://www.youtube.com/watch?v=PptCfa73Y1k) livestream.

The stream takes place [on Youtube](https://www.youtube.com/watch?v=PptCfa73Y1k) on **Thursday, July 15** at **5pm Berlin | 8am San Francisco**.

<hr /><em>This discussion was created from the release <a href='https://github.com/prisma/prisma/releases/tag/2.28.0'>2.28.0</a>.</em>

**Comment by stnwk** [comment]
<img width="776" alt="image" src="https://user-images.githubusercontent.com/2981250/127207364-6e64e194-a381-4586-a399-5b3e4805ccd0.png">

I can't submit the form at https://pris.ly/prisma-data-proxy because if I don't want the newsletter it tells me to accept the terms & conditions lol

**Reply by janpio** [reply]
That is unexpected. Forwarded it to the right people and will update when fixed.

**Reply by albertoperdomo** [reply]
Hello @stnwk,

It was a mistake. It's fixed now,  so you should be able to sign up now without opting in for the newsletter.

Thanks for reporting!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
